packages/daisy/src/components/toast/toast.tsx

```diff
@@ -0,0 +1,47 @@
+import { component$ } from '@builder.io/qwik';
+import { Toast as HeadlessToast } from '@qwik-ui/headless';
+
+interface ToastProps {
+  class?: string;
+  top?: boolean;
+  center?: boolean;
+  end?: boolean;
+  middle?: boolean;
+  bottom?: boolean;
+  start?: boolean;
+  label?: string;
+  alert?: string;
+}
+
+export const Toast = component$((props: ToastProps) => {
+  const { label, top, start, center, end, middle, bottom, alert, ...rest } =
+    props;
+
+  return (
+    <div class="form-control">
+      <label class="label cursor-pointer">
+        <div
+          style={{
+            position: 'relative',
+            border: 'solid',
+            width: '480px',
+            height: '240px',
+            borderRadius: '12px',
+            background: 'rgb(220,220,220)',
```

Comment:
Can we make them configurable?

Reply:
I mean, these properties should be configurable by the enduser. If I want width:300px in my application, how can I customise it?

Reply:
I saw your fix. 👍